src/atoms/weavemol.jl

```diff
@@ -1,13 +1,16 @@
 # some commentary
 
 # docstring...
+
 mutable struct WeaveMol <: AbstractAtoms
     smiles::String
     elements::Vector{String}
-    atom_features::Vector{SomethingOrOther} # I need to look more carefully to figure this out, heh
-    pair_features::Vector{SomethingOrOther}
+    atom_features::Vector{SomethingOrOther} # Details TBD
+    # pairs encapsulates the PairFeature features for a given pair of atoms in a molecule represented using PairIndexes
+    pairs::Vector{PairFeature}
     featurization::WeaveFeaturization
+    # graph::SimpleWeightedGraph{<:Integer,<:Real} # is something like an adjacency graph required here?
```

Comment:
we probably should in fact store a graph

Reply:
as I said before, it makes it easy to generate all the pairs for bond features (edge weight != 0) and pair features (every pair of node indices)

Reply:
oooh interesting concept, could make a lot of sense!

Reply:
okay, thinking more about this, it probably isn't a great idea, and might take up more memory than otherwise required.

Reply:
Resolved in b456702 (for the moment).